host/3.0/core-tools-publish.yml

```diff
@@ -343,38 +385,72 @@ stages:
     
       - bash: |
           set -e
-          docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet3-core-tools
+          docker pull $SOURCE_REGISTRY/dotnet:$(PrivateVersion)-dotnet-isolated5.0-core-tools
```

Comment:
I think these would be "$SOURCE_REGISTRY/**dotnet-isolated**:$(PrivateVersion)-dotnet-isolated5.0-core-tools". Same throughout this file.

Reply:
Very good catch, I fixed wrong place.  Could you review this place again? @ankitkumarr